metadata.json

```diff
@@ -0,0 +1,5 @@
+{
+    "citation": "Chechuro 2021",
```

Comment:
Please provide the full title in the citation. Use Forthcoming, if data are not yet published.